eoscostarica/eos-rate
# feat(compare): basic compare tool functionality

### Steps for testing
- Go to block producers page
- A big dark box should appear above the list of block producers, with the title compare tool.
- Go ahead and click the `+` sign on any block producer, and then click a second one.
- Both block producers should appear in the compare tool on the same radar graph.
- Click on the switch on the bottom right corner, it should change to a horizontal slider to show individual block producers.

![image](https://user-images.githubusercontent.com/349542/47962983-e9bd3f00-e01c-11e8-8d4c-82f76f45e11e.png)
